src/app/recommended-report-card/recommended-report-card.component.ts

```diff
@@ -57,7 +57,7 @@ export const TOXICITY_RANGE_TEMPLATES: RecommendedReportTemplate[] = [
     },
     name: 'Likely',
     description: 'Likely harmful',
-    icon: 'high_priority_icon.svg',
+    icon: 'red_priority_icon.svg',
```

Comment:
Lets keep the original names (`high_priority`, `unknown_priority`, etc.) and rename the icons instead.

Reply:
It should be 1440?